docs/learn/court.md

```diff
@@ -6,8 +6,298 @@ title: Decentralized Court
+consideration of all `delegations`. If there are no valid jurors inside the
+`delegations` list, the delegator is removed from the court pool (error inside
+the code: `SelectionError::NoValidDelegatedJuror`).
+
+For delegations a vote weight (code reference: `SelectionAdd::DelegationWeight`)
+goes to a random and valid juror from the `delegations` list, but the delegator
+risks (`slashable` and code reference `SelectionAdd::DelegationStake`) the
+`MinJurorStake` associated to the vote weight. If the juror makes bad decisions,
+the delegator loses the selected `MinJurorStake`.
+
+### Voting at predefined Time Points
+
+Jurors are requested to vote in a periodic interval (`RequestInterval`) at a
+known request block in the future. This ensures that jurors only need to check
+at predefined times if they need to take action. If there wasn’t this concept of
+predefined requests, the jurors would have needed to check in a much smaller
+time interval if they are selected in court cases.
+
+### Commitment Voting
+
+The voting system uses a commit-reveal scheme, which is required to prevent
+jurors from simply voting for the obvious plurality decision. There are voting,
+aggregation, and appeal phases. During the voting phase, jurors cast their votes
+as encrypted hashes (extrinsic `vote`), which must later be revealed (extrinsic
+`reveal_vote`) as raw information. If a juror fails to vote or reveal their
+vote, they lose their stake for that specific court case. The encrypted hash
+consists of a BlakeTwo256 hash of the juror account, the vote item (outcome),
+and a salt. The salt is a hash derived from the juror's signature of the
+specific court ID. Without a salt, a malicious actor could try every vote item
+to obtain raw information. If a juror's salt is known before the voting phase
+ends, they could be exposed and penalized (extrinsic `denounce_vote`) by those
+aware of the salt. This adds a layer of protection against cheating and
+increases trust in the system.
+
+### Appeals
+
+After the aggregation phase ends and all jurors have had the chance to make
+their votes public, anyone can appeal the plurality decision (during the appeal
+period). This triggers a new court round with more requested jurors and
+delegators' stakes or (in case after the last round) allows a global dispute to
+take over. If nobody appeals, the court resolves based on the plurality decision
+of the last court round. Finally, the losers must pay the winners proportionally
+to their selected stakes for the specific court case (extrinsic
+`reassign_court_stakes`).
+
+In order to make an appeal, the caller of `appeal` has to reserve a bond. The
+cost of an appeal is calculated as following:
+`cost of appeal = AppealBond * 2^(appeal_number + 1)`
+
+1. first appeal cost: `2000 ZTG (AppealBond) * 2^1 = 4000 ZTG`
+2. second appeal cost: `2000 ZTG (AppealBond) * 2^2 = 8000 ZTG`
+3. third appeal cost: `2000 ZTG (AppealBond) + 2^3 = 16000 ZTG`
```

Comment:
First letter needs to be capitalized here.

Reply:
https://github.com/zeitgeistpm/documentation/pull/81/commits/a19875040e67869b6cc091dad09edd5163eef651